woocommerce-services.php

```diff
@@ -282,14 +276,9 @@ public static function get_wcs_admin_style_url() {
 			}
 		}
 
-		function wpcom_static_url( $file ) {
-			$i   = hexdec( substr( md5( $file ), -1 ) ) % 2;
-			$url = 'http://s' . $i . '.wp.com' . $file;
-			return set_url_scheme( $url );
-		}
-
 		public function __construct() {
 			$this->wc_connect_base_url = self::get_wc_connect_base_url();
+			add_action( 'plugins_loaded', array( $this, 'jetpack_on_plugins_loaded' ), 1 );
```

Comment:
It's important that this code is executed with priority `1` - I banged my head for a while trying to figure out why it wasn't working before 😄

Reply:
https://github.com/Automattic/woocommerce-payments/pull/637/files#diff-ad243b7f0e0ae2cc54963fec106a5c54ece9c5bb8b272acaeece1836d71418a2R36